src/dodal/devices/i18/KBMirror.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+
+from ophyd_async.core import AsyncStatus, StandardReadable
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+@dataclass
+class XYPosition:
+    x: float
+    y: float
```

Comment:
for consistency with https://github.com/DiamondLightSource/dodal/blob/9cb9596dac6cb1bf89a4f2ea3d1ba4171275b9c4/src/dodal/devices/aperturescatterguard.py#L135

Please use a BaseModel

Reply:
fair enough

Reply:
added